mqtt-cs/src/main/java/org/apache/rocketmq/mqtt/cs/session/Session.java

```diff
@@ -142,9 +142,7 @@ public Map<Subscription, Map<Queue, QueueOffset>> offsetMapSnapshot() {
             }
             Map<Queue, QueueOffset> queueMap = new HashMap<>(8);
```

Comment:
the size 8 can be replaced by offsetMap.get(queueName).size()

Reply:
> the size 8 can be replaced by offsetMap.get(queueName).size()

good catch👍, done